src/components/Header/Header.jsx

```diff
@@ -1,41 +1,72 @@
 import { Link, useNavigate } from 'react-router-dom'
 import "./Header.css"
 import { useAuth } from '../../contexts/AuthContext'
+import { useCart } from '../../contexts/CartContext'
+import { useWishlist } from '../../contexts/WishlistContext'
 
 export default function Header() {
-    const {loggedIn, logoutHandler} = useAuth()
     const navigate=useNavigate()
-    console.log(loggedIn)
+    const {loggedIn, logoutHandler} = useAuth()
+    const {wishlistCount} = useWishlist()
+    const {cartCount}=useCart()
 
     function handleLogin(e){
         loggedIn  ?  logoutHandler() : navigate("/")
     }
 
-  return (
-    <header className="header shadow">
-        <Link to="/">
-            <img className="main-logo" src={process.env.PUBLIC_URL + 'assets/mainLogo.png'}/>
-        </Link>
-        <div className="search-bar">
-            <Link to="/products" className="search-submit">
-                <span className="material-icons material-icons-outlined">
-                    search
-                </span>
+    return (
+        <header className="header shadow">
+            <Link to="/">
+                <img className="main-logo" alt="the-paper-dream-logo" src={process.env.PUBLIC_URL + 'assets/mainLogo.png'}/>
             </Link>
-            <input type="text" className="search-input" placeholder="Search"/>
-        </div>
-        <ul className="header-actions">
-            <li>
-                <Link to={loggedIn ? "/" : "/login"} className="header-ctas" onClick = {handleLogin}>
-                    <h4>{ loggedIn ? "Logout" : "Login"}</h4>
+            <div className="search-bar">
+                <Link to="/products" className="search-submit">
+                    <span className="material-icons material-icons-outlined">search</span>
                 </Link>
-            </li>
-            <li>
-                <Link to="/signup" className="header-ctas">
-                    <h4>Sign Up</h4>
-                </Link>
-            </li>
-        </ul>
-    </header>
-  )
+                <input type="text" className="search-input" placeholder="Search"/>
+            </div>
+            <ul className="header-actions">
+                {loggedIn ? 
```

Comment:
noticed that if  user not logged in you dont want to render anything for that so you have given empty string so instead you can use short circuiting....please correct me if my understanding over code, if not correct